homeassistant/components/cover/mqtt.py

```diff
@@ -44,6 +57,16 @@
     vol.Optional(CONF_STATE_OPEN, default=STATE_OPEN): cv.string,
     vol.Optional(CONF_STATE_CLOSED, default=STATE_CLOSED): cv.string,
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
+    vol.Optional(CONF_TILT_CLOSED_POSITION,
+                 default=DEFAULT_TILT_CLOSED_POSITION): cv.string,
+    vol.Optional(CONF_TILT_OPEN_POSITION,
+                 default=DEFAULT_TILT_OPEN_POSITION): cv.string,
+    vol.Optional(CONF_DEVICE_CLASS,
+                 default=DEFAULT_DEVICE_CLASS): cv.string,
+    vol.Optional(CONF_TILT_MIN,
+                 default=DEFAULT_TILT_MIN): cv.string,
```

Comment:
Do you have a use case for configurable min/max? [The cover component](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/cover/__init__.py#L66) is already built around 0-100, so we should just continue with that unless it's necessary.